src/pharmpy/workflows/model_database/baseclass.py

```diff
@@ -37,6 +37,34 @@ def store_local_file(self, model, path):
         """
         pass
 
+    @abstractmethod
+    def store_optional_local_file(self, model, path):
```

Comment:
What does "optional" in the name refer to? Could the `store_local_file` method have been used?

Reply:
All files stored by `*_optional_*` methods are not written to disk in the `LocalDirectoryDatabase` implementation. Could have a better naming.